plugins/extractors/postgres/postgres.go

```diff
@@ -25,12 +26,11 @@ import (
 //go:embed README.md
 var summary string
 
-var defaultExcludes = []string{"information_schema", "root", "postgres"}
+var defaultDBList = []string{"information_schema", "root", "postgres"}
 
 // Config holds the set of configuration options for the extractor
 type Config struct {
 	ConnectionURL string `mapstructure:"connection_url" validate:"required"`
-	Exclude       string `mapstructure:"exclude"`
```

Comment:
I will restore this one, but wanted to know if it something we want to allow with every other extractor as well.?? Or if there is some reason to have it only for Postgres. 

cc @StewartJingga @scortier